CI-CD/production-deployment-with-tests-java/ext/test-config.json

```diff
@@ -1,5 +1,5 @@
 {
     "localEndpoints": {
-        "default" : "http://localhost:8080/"
+        "default" : "https://default.myapp.mytenant.aws-us-east-1c.dev.z.vespa-app.cloud/"
```

Comment:
??

Reply:
I could not find doc on the config file content - but how to provide the endpoint if not like this?